prac_06/guitars.py

```diff
@@ -0,0 +1,30 @@
+"""
+Guitars
+Estimate: 45 minutes
+Actual: 55 minutes
+"""
+
+from prac_06.guitar import Guitar
+
+guitars = []
+
+print("My guitars!")
+name = input("Name: ").title()
+while name != "":
+    year = int(input("Year: "))
```

Comment:
I think that you can comment out these inputs because the instruction told that we can use .append() instead of getting input from user.